ERROR - 2022-10-02 16:56:09 --> mysqli_sql_exception: Table 'pos_db.file' doesn't exist in C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `fi...', 0)
#1 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fi...')
#2 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fi...')
#3 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fi...', Array, false)
#4 C:\xampp_v8\htdocs\crud_ci4\app\Controllers\Form.php(39): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(904): App\Controllers\Form->store()
#6 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Form))
#7 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp_v8\htdocs\crud_ci4\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-10-02 16:57:10 --> mysqli_sql_exception: Table 'pos_db.file' doesn't exist in C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `fi...', 0)
#1 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fi...')
#2 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fi...')
#3 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fi...', Array, false)
#4 C:\xampp_v8\htdocs\crud_ci4\app\Controllers\Form.php(39): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(904): App\Controllers\Form->store()
#6 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Form))
#7 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp_v8\htdocs\crud_ci4\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-10-02 16:58:00 --> mysqli_sql_exception: Table 'pos_db.file' doesn't exist in C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `fi...', 0)
#1 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fi...')
#2 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fi...')
#3 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fi...', Array, false)
#4 C:\xampp_v8\htdocs\crud_ci4\app\Controllers\Form.php(39): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(904): App\Controllers\Form->store()
#6 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Form))
#7 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp_v8\htdocs\crud_ci4\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-10-02 16:59:20 --> mysqli_sql_exception: Table 'pos_db.file' doesn't exist in C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `fi...', 0)
#1 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fi...')
#2 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fi...')
#3 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fi...', Array, false)
#4 C:\xampp_v8\htdocs\crud_ci4\app\Controllers\Form.php(39): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(904): App\Controllers\Form->store()
#6 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Form))
#7 C:\xampp_v8\htdocs\crud_ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp_v8\htdocs\crud_ci4\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
